server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponse.java

```diff
@@ -141,16 +160,17 @@ public boolean equals(Object o) {
     public String toString() {
         return "PulseResponse{" +
                 "id=" + id +
+                ", internal_score" + internalScore +
+                ", external_score" + externalScore +
```

Comment:
`internalScore` and `externalScore` ...these are the Java fields, not the db columns.

Reply:
Adjusted